dotnet/src/SemanticKernel.UnitTests/TemplateEngine/Blocks/VarBlockTests.cs

```diff
@@ -37,33 +37,33 @@ public void ItIgnoresSpacesAround()
     }
 
     [Fact]
-    public void ItRendersToEmptyStringWithoutArgument()
+    public void ItRendersToNullWithNoArgument()
     {
         // Arrange
-        var target = new VarBlock("  $var \n ");
+        var target = new VarBlock("$var");
 
         // Act
-        var result = target.Render(null);
+        var result = target.Render(new KernelArguments());
 
         // Assert
-        Assert.Equal(string.Empty, result);
+        Assert.Null(result);
     }
 
     [Fact]
-    public void ItRendersToEmptyStringIfArgumentIsMissing()
+    public void ItRendersToNullWithNullArgument()
     {
         // Arrange
-        var target = new VarBlock("  $var \n ");
+        var target = new VarBlock("$var ");
```

Comment:
Can't understand what this test is really doing.

`$var` isnt supposed to be a parameter named `var` ? 

If you create an argument `foo` and pass to the target, it means that if you have a variable foo, it won't be case to var, is that it?

Reply:
Right, it was typo that is fixed now. Thanks for catching.